plannerbenchmark/postProcessing/metrics.py

```diff
@@ -40,42 +62,67 @@ def computeMetric(self, data):
 
 
 class TimeToReachGoalMetric(Metric):
+    """Metric to compute the time it took to reach the goal.
+
+    Requires the threshold, `des_distance`.
+    """
+
     def computeMetric(self, data):
-        m = self._params['m']
+        m = self._params["m"]
         fks = np.stack([data[name] for name in self._measNames[:m]]).T
         goal = np.stack([data[name] for name in self._measNames[m:-1]]).T
         t = data[self._measNames[-1]]
         des_distance = self._params["des_distance"]
         distances = computeDistances(fks, goal)
-        minDistance = np.min(distances)
         indices = np.where(distances < des_distance)
         if indices[0].size == 0:
-            return {'short': -1, 'flag': -1}
+            return {"short": -1, "flag": -1}
         else:
-            return {'short': float(t[np.min(indices)]), 'flag': 0}
+            return {"short": float(t[np.min(indices)]), "flag": 0}
 
 
 class IntegratedErrorMetric(Metric):
+    """Metric to compute the integrated deviation error from reference trajectory.
+
+    Requires the threshold, `des_distance`.
+    This metric computes the averaged deviation error from the first time the
+    threshold was reached.
+    """
+
     def computeMetric(self, data):
-        m = self._params['m']
-        des_distance = self._params['des_distance']
+        m = self._params["m"]
+        des_distance = self._params["des_distance"]
         fks = np.stack([data[name] for name in self._measNames[:m]]).T
         goal = np.stack([data[name] for name in self._measNames[m:-1]]).T
         t = np.array(data[self._measNames[-1]])
```

Comment:
Is 't' here timetoreachgoal or a different form of time measurement? A more representative name for 't' will render clarity.